### PR TITLE
Blog post for the wrapper 2022.

### PR DESCRIPTION
**Pull request by fcorti**
Signed-off-by: Francesco Corti <francescoc@spotify.com>

## Hey, I just made a Pull Request!

Sorry but prettier did not work for me and I did not execute it.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [*] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [*] Screenshots attached (for UI changes)
- [*] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by fcorti** [comment]
I managed to execute the prettier (`yarn prettier --write .`) from the root of the project instead the `microsite` folder.
It worked fine but it did not change any content so I did not commit any new content.

Please let me know if I should do something different for the future.

Thanks.

**Comment by marcuseide** [comment]
Hey @fcorti!
Did you run the `prettier` command in the `microsite` folder?

Try:
```bash
cd microsite
yarn install
yarn prettier --write . 
yarn prettier:check
```
For the docs quality check I think you need to add the word `Deutsche` to `.github/vale/Vocab/Backstage/accept.txt`.

**Comment by fcorti** [comment]
You are my hero @marcuseide!
I'm was missing the yarn install at the microsite level (I did it at the root level of the project).

**Comment by fcorti** [comment]
Ok, now I also added the word `Deutsche` to the `accept.txt` file.

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Backstage Wrapped 2022
+author: Francesco Corti, Suzanne Daniels
+authorURL: https://github.com/fcorti
+authorImageURL:
+---
+
+If 2021 was the year of growth for the Backstage project and community, 2022 can be easily defined as the year of maturity. Both the project and the community continued to grow at an incredible pace and the platform is now more stable, more secure, and more powerful than ever.
+
+Our community is full of people from across the globe that brings incredible engagement and positive vibes to the project. And we finally got to experience this for the first time IRL during [BackstageCon](https://www.youtube.com/playlist?list=PLj6h78yzYM2OKySsTuiip3BqmdYZQRnSf), the very first in-person conference fully dedicated to Backstage, where hundreds of enthusiasts, the maintainers, adopters, and partners joined the event confirming the great momentum and the traction in the market.
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:10]:
```suggestion
Our community is full of people from across the globe who bring incredible engagement and positive vibes to the project. And we finally got to experience this for the first time in real life during [BackstageCon](https://www.youtube.com/playlist?list=PLj6h78yzYM2OKySsTuiip3BqmdYZQRnSf), the very first in-person conference fully dedicated to Backstage, where hundreds of enthusiasts, the maintainers, adopters, and partners joined the event confirming the great momentum and the traction in the market.
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+But before anticipating too much, let’s share some stats of this amazing 2022.
+
+<!--truncate-->
+
+## We grew … a lot!
+
+Last year we celebrated 75 [public adopters](https://github.com/backstage/backstage/blob/master/ADOPTERS.md) and today we have over 600 adopters, confirming the interest around Backstage across all the industries, sectors, and dimensions. Companies like [Toyota North America](https://aws.amazon.com/solutions/case-studies/toyota-ecs-case-study/), Deutsche Bank, Mercedes-Benz.io, Procter & Gamble, Siemens, LinkedIn, and many more joined the community this year and are actively contributing in growing the product and its maturity.
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:18]:
```suggestion
Last year we celebrated 75 [public adopters](https://github.com/backstage/backstage/blob/master/ADOPTERS.md) and today we have over 600 adopters, confirming the interest around Backstage across all the industries, sectors, and dimensions. Companies like [Toyota North America](https://aws.amazon.com/solutions/case-studies/toyota-ecs-case-study/), Deutsche Bank, Mercedes-Benz, Procter & Gamble, Siemens, LinkedIn, and many more joined the community this year and are actively contributing in growing the product and its maturity.
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+![Accelerated growth of Public Backstage Adopters, from January 2021–December 2022.](assets/2022-12-19/public-backstage-adopters-2022.png)
+
+Not only are our adopters growing at an exponential rate, but so too are our contributors. Our [Discord channel](https://discord.gg/backstage-687207715902193673) is a perfect example of the worlds of our adopters and contributors melding. On average, at any given hour of the day or night, there are more than 500 people online available to talk and provide support on any Backstage topic. Discord discussions run a full gamut of topics – from technical support questions to best practices for onboarding engineers to Backstage – basically, anything and everything an adopter may be interested to know and learn. At the very beginning of 2022, we “only” had 4k Discord members. Today, we have a growing Discord channel of 8.2k.
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:22]:
```suggestion
Not only are our adopters growing at an exponential rate, but so too are our contributors. Our [Discord server](https://discord.gg/backstage-687207715902193673) is a perfect example of the worlds of our adopters and contributors melding. On average, at any given hour of the day or night, there are more than 500 people online available to talk and provide support on any Backstage topic. Discord discussions run a full gamut of topics – from technical support questions to best practices for onboarding engineers to Backstage – basically, anything and everything an adopter may be interested to know and learn. At the very beginning of 2022, we “only” had 4k Discord members. Today, we have a growing membership of 8.2k.
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Before we go any further, let’s take a quick look at some more numbers, mainly coming from the [Official Github repository](https://github.com/backstage/backstage). This year…
+
+- Our contributors grew from a group of 500 to 900+
+- Contributions to the project increased from 9k to 15k+
+- The number of forks shot up from 2k to 3k+
+- Backstage stars increased from 14k to almost 20k
+- And lastly, we welcomed over 550 adopters into our community.
+
+All of these people driving all enormous growth and maturity for the project is represented in the chart below, which shows cumulative activities per month for Backstage:
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:36]:
```suggestion
All of these people driving the enormous growth and maturity of the project are represented in the chart below, which shows cumulative activities per month for Backstage:
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+![Main Github metrics for Backstage.io.](assets/2022-12-19/github-metrics-for-backstage-io.png)
+
+Wow… that’s a lot of activity. We’ve generated this animation showing all of your contributions to the Backstage repository in 2022, look at this project grow:
+
+<iframe width="780" height="440" src="https://youtu.be/GoWBvHpH5Cg" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
+
+## But wait… there’s more
+
+The growth of the community is so exciting to see. And with all this community growth, comes growth and maturity of the product, making Backstage better able to support adopters in their developer experience challenges. The Backstage open source project has almost [100 plugins](https://backstage.io/plugins), each one implementing a Backstage with a tool or service to improve not only adopters’ Backstage apps, but their unique developer experiences. And almost each week we are aware of a new one being contributed, that is amazing.
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:46]:
```suggestion
The growth of the community is so exciting to see. And with all this community growth, comes growth and maturity of the product, making Backstage better able to support adopters in their developer experience challenges. The Backstage open source project has almost [100 plugins](https://backstage.io/plugins), each one amending Backstage with a tool or service to improve not only adopters’ Backstage apps, but their unique developer experiences. And almost every week we learn of a new one being contributed, which is amazing.
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+The Backstage platform itself is leaving 2022 more mature than ever, thanks to a few of these amazing milestones:
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:48]:
```suggestion
The Backstage platform itself is leaving 2022 more mature than ever, thanks to milestones such as these:
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+And like every mature project, now Backstage has also a mascot: Bowie the Beaver!
+
+![Bowie the Beaver.](assets/2022-12-19/bowie-the-beaver.png)
+
+Additionally, we launched a new [Special Interest Groups (SIGs)](https://github.com/backstage/community/tree/main/sigs) initiative this year, following the lead of Kubernetes. We believe Backstage SIGs will better coordinate contributions, grow the platform cohesively with the open source community, and allow contributors to dig even deeper as a community member. At this moment in time there two SIGs in place:
+
+- The [Catalog SIG](https://github.com/backstage/community/blob/main/sigs/sig-catalog/README.md), covering all aspects of the systems and components related to the Software Catalog.
+- The [Adoption SIG](https://github.com/backstage/community/blob/main/sigs/sig-adoption/README.md), covering all aspects of the adoption journey of Backstage.
+
+Our plan is to open more SIGs in the 2023 to engage more with the community and to keep driving towards cohesive growth for the community and project.
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:64]:
```suggestion
Our plan is to open more SIGs in 2023 to engage more with the community and to keep driving towards cohesive growth for the community and project.
```

File: microsite/blog/2022-12-19-backstage-wrapped-2022.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+Last but not least, we cannot miss the first ever conference fully dedicated to Backstage: BackstageCon. What was special there? SO much. But in particular, two things: seeing everyone in real life and the incredible mood and conversations between maintainers, adopters, partners and enthusiasts. It’s hard to emanate just how positive and amazing the vibes were in a blog, but you can read our recap [here](https://backstage.io/blog/2022/10/28/backstagecon-kubecon-2022) or watch the sessions [here](https://www.youtube.com/playlist?list=PLj6h78yzYM2OKySsTuiip3BqmdYZQRnSf). And in the meantime, here are some great numbers from the event:
```
Comment (freben) [inline microsite/blog/2022-12-19-backstage-wrapped-2022.md:66]:
```suggestion
Last but not least, we mustn't overlook the first ever conference fully dedicated to Backstage: BackstageCon. What was special there? SO much. But in particular two things: seeing everyone in real life and the incredible mood and conversations between maintainers, adopters, partners and enthusiasts. It’s hard to express just how positive and amazing the vibes were in a blog, but you can read our recap [here](https://backstage.io/blog/2022/10/28/backstagecon-kubecon-2022) or watch the sessions [here](https://www.youtube.com/playlist?list=PLj6h78yzYM2OKySsTuiip3BqmdYZQRnSf). And in the meantime, here are some great numbers from the event:
```

**Review by freben — APPROVED** [review]
Some wording adjustment, but pre-approving to unblock!

**Pull request merged**